src/lib/services/access-service.ts

```diff
@@ -643,6 +649,17 @@ export class AccessService {
                 );
             }
         }
+        const addedPermissions = await this.store.getPermissionsForRole(
+            newRole.id,
+        );
```

Comment:
Currently, permissions can come in any shape from the UI (i.e. only send ids) and we'll adapt, but we just want the names in the event log. This also applies to other parts of this PR